install.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+echo 'vm.max_map_count=262144' | sudo tee --append /etc/sysctl.d/99-elasticsearch.conf
+sudo sysctl -p /etc/sysctl.d/99-elasticsearch.conf
+
+mkdir -p  /tmp/_working_dir
+
+# mkdir ~/.kube
+# sudo cp /etc/origin/master/admin.kubeconfig ~/.kube/config
+# sudo chown origin:origin ~/.kube/config
+
+oc create -n openshift-logging -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml
+oc create -n openshift-logging -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml
+oc create -n openshift-logging -f https://raw.githubusercontent.com/openshift/cluster-logging-operator/01cbb3c3c4fb3c34fc7204497747038c4ff22779/deploy/cr.yaml
+
+oc create -f hack/cr.yaml
+
+REPO_PREFIX=openshift/ IMAGE_PREFIX=origin- OPERATOR_NAME=cluster-logging-operator WATCH_NAMESPACE=openshift-logging KUBERNETES_CONFIG=~/.kube/config ELASTICSEARCH_IMAGE=docker.io/openshift/origin-logging-elasticsearch5 ./main
+
+ALERTS_FILE_PATH=files/prometheus_alerts.yml RULES_FILE_PATH=files/prometheus_rules.yml PROXY_IMAGE=quay.io/openshift/origin-oauth-proxy:v4.0.0 REPO_PREFIX=openshift/ IMAGE_PREFIX=origin- OPERATOR_NAME=elasticsearch-operator WATCH_NAMESPACE=openshift-logging KUBERNETES_CONFIG=~/.openshift/aws/auth/kubeconfig LOG_LEVEL=debug go run cmd/elasticsearch-operator/main.go
+
+docker.io/openshift/oauth-proxy:v1.1.0
```

Comment:
?

Reply:
was the rest of this file intended to go into a readme or comment?

Reply:
I committed the file accidentally - I will remove it.